pkg/config/defaults.go

```diff
@@ -47,6 +47,9 @@ func setDefaults(local bool) {
 	viper.SetDefault("auth.ui_config.login_failed_message", "The credentials don't match.")
 	viper.SetDefault("auth.ui_config.login_cookie_names", "internal_auth_session")
 
+	viper.SetDefault("auth.remote_authenticator.default_user_group", "Viewers")
+	viper.SetDefault("auth.remote_authenticator.request_timeout", 10*time.Second)
```

Comment:
note that the configuration documentation declares the default request timeout is 7s and not 10s.

Reply:
good catch thanks 👍